### PR TITLE
OSPF v2/v3 router priority (CL, EOS, FRR, IOS, NXOS)

### PR DESCRIPTION
**Pull request by ipspace**
Includes integration tests and minor tweaks to the OSPF validation plugins

### CODE REVIEW

File: tests/integration/ospf/ospfv3/31-priority.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+message: |
+  This lab tests the OSPF interface timers. The device under test must establish
```
Comment (jbemmel) [inline tests/integration/ospf/ospfv3/31-priority.yml:3]: OSPF interface priority

**Review by jbemmel — COMMENTED** [review]
VRF test should be revised

**Comment by ipspace** [comment]
> VRF test should be revised

Yeah, I have no idea why it works ;)

**Comment by jbemmel** [comment]
#1667 has proposed changes - I think it's better to check the neighbor priority directly

**Comment by ipspace** [comment]
> #1667 has proposed changes - I think it's better to check the neighbor priority directly

Absolutely. I missed that bit of information.

I will cherry-pick the test into 'dev' branch while we're waiting for @ssasso to review #1667.

**Comment by ipspace** [comment]
> #1667 has proposed changes - I think it's better to check the neighbor priority directly

FWIW, it would have been so much easier (for me) if you also fixed the OSPFv3 test 😜

**Pull request merged**